### PR TITLE
Documentation/58141 Document Nextcloud GroupFolder encryption activation command

### PR DESCRIPTION
**Pull request by dominic-braeunlein**
# Ticket
[#58141](https://community.openproject.org/projects/openproject/work_packages/58141)

# What are you trying to accomplish?
This PR adds a description for Nextcloud admins that face the problem that some files are not encrypted on Nextcloud

### CODE REVIEW

**Review by Kharonus — APPROVED** [review]
LGTM.

But it raises the question, if actually our integration should be able to detect server-side encryption automatically and set this configuration. On the other hand, configuring 3rd party apps generally is a bad idea from any plugin.

**Comment by dominic-braeunlein** [comment]
> if actually our integration should be able to detect server-side encryption automatically and set this configuration.

We are investigating on a detection of the encryption and displaying a warning in the settings
https://community.openproject.org/projects/nextcloud-integration/work_packages/57970/activity#activity-10

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 1]
```diff
@@ -373,6 +373,16 @@ sudo systemctl restart apache2
 
 If that also not work please check the [instructions for setting up pretty URLs in Nextcloud](https://docs.nextcloud.com/server/22/admin_manual/installation/source_installation.html#pretty-urls). This usually also resolves the issue of stripped `Authorization` headers.
 
+#### Files are not encrypted when using Nextcloud server-side encryption
+
+If your Nextcloud server uses server-side encryption, please note that the GroupFolder encryption needs to be manually enabled.
```
Comment (MayaBerd) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
> [!NOTE]
If your Nextcloud server uses server-side encryption, the GroupFolder encryption needs to be manually enabled. 
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 2]
```diff
@@ -373,6 +373,17 @@ sudo systemctl restart apache2
 
 If that also not work please check the [instructions for setting up pretty URLs in Nextcloud](https://docs.nextcloud.com/server/22/admin_manual/installation/source_installation.html#pretty-urls). This usually also resolves the issue of stripped `Authorization` headers.
 
+#### Files are not encrypted when using Nextcloud server-side encryption
+
+> [!NOTE]
+If your Nextcloud server uses server-side encryption, the GroupFolder encryption needs to be manually enabled.
```
Comment (MayaBerd) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
> If your Nextcloud server uses server-side encryption, the GroupFolder encryption needs to be manually enabled. 
```

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 3]
```diff
@@ -373,6 +373,17 @@ sudo systemctl restart apache2
 
 If that also not work please check the [instructions for setting up pretty URLs in Nextcloud](https://docs.nextcloud.com/server/22/admin_manual/installation/source_installation.html#pretty-urls). This usually also resolves the issue of stripped `Authorization` headers.
 
+#### Files are not encrypted when using Nextcloud server-side encryption
+
+> [!NOTE]
+> If your Nextcloud server uses server-side encryption, the GroupFolder encryption needs to be manually enabled. 
+This is relevant for automatically managed project folders, as the GroupFolder app is used in these cases.
```
Comment (MayaBerd) [inline docs/system-admin-guide/integrations/nextcloud/README.md]:
```suggestion
> This is relevant for automatically managed project folders, as the GroupFolder app is used in these cases. 
```

**Pull request merged**